docker/Dockerfile.x86_64-unknown-linux-gnu.centos

```diff
@@ -26,6 +26,9 @@ COPY --from=0 /qemu /qemu
 
 COPY linux-runner base-runner.sh /
 
+# NOTE: this target is only supported on linux/amd64 hosts,
```

Comment:
I think we can actually remove this file since it's never used if `native.centos` is used instead.